src/main/java/graphql/schema/idl/SchemaPrinter.java

```diff
@@ -241,8 +285,16 @@ public Options includeDirectives(boolean flag) {
          *
          * @return new instance of options
          */
-        public Options includeDirectives(Predicate<GraphQLDirective> includeDirective) {
-            return new Options(this.includeIntrospectionTypes, this.includeScalars, this.includeSchemaDefinition, this.includeDirectiveDefinitions, this.useAstDefinitions, this.descriptionsAsHashComments, includeDirective, this.includeSchemaElement, this.comparatorRegistry);
```

Comment:
BREAKING CHANGE - again we could do work to make this NOT a breaking change BUT I dont think its worth it.

Thoughts?